matchzoo/contrib/models/lbmp.py

```diff
@@ -0,0 +1,131 @@
+"""LBMP model."""
+import typing
+import keras
+
+from matchzoo.engine import hyper_spaces
+from matchzoo.engine.param_table import ParamTable
+from matchzoo.engine.param import Param
+from matchzoo.engine.base_model import BaseModel
+
+
+class LBMP(BaseModel):
+    """
+    LBMP model.
+
+    Examples:
+        >>> model = LBMP()
+        >>> model.guess_and_fill_missing_params(verbose=0)
+        >>> model.params['embedding_input_dim'] = 10000
+        >>> model.params['embedding_output_dim'] = 100
+        >>> model.params['embedding_trainable'] = True
+        >>> model.params['leaky_relu_alpha'] = 0.1
+        >>> model.params['mlp_num_layers'] = 3
+        >>> model.params['mlp_num_units'] = [600, 600]
+        >>> model.params['lstm_num_units'] = 600
+        >>> model.params['dropout_rate'] = 0.1
+        >>> model.build()
+
+    """
+
+    @classmethod
+    def get_default_params(cls) -> ParamTable:
+        """:return: model default parameters."""
+        params = super().get_default_params(with_embedding=True)
+        params['optimizer'] = 'adam'
+        params.add(Param(name='leaky_relu_alpha', value=0.1,
```

Comment:
`alpha` to be consistent with Keras

Reply:
ok, I have renamed the 'alpha' parameter.